docs/pages/guides/using-sentry.mdx

```diff
@@ -18,6 +18,16 @@ It notifies you of exceptions or errors that your users run into while using you
 
 <PlatformsSection title="Platform compatibility" android emulator ios simulator web />
 
+## What you'll learn
+
+This guide covers three main aspects of integrating Sentry with your Expo projects:
+
+1. [Installing and configuring Sentry](#install-and-configure-sentry) in your React Native application
+2. Using Sentry with Expo services:
+   - [EAS Build](#usage-with-eas-build) for building your application
+   - [EAS Update](#usage-with-eas-update) for over-the-air updates
+3. [Setting up the Sentry-Expo integration](#sentry-integration-with-eas) to view crash reports and session replays directly in your EAS dashboard
```

Comment:
```suggestion
1. [Install and configure Sentry](#install-and-configure-sentry) in your React Native app
2. Using Sentry with EAS:
   - [EAS Build](#usage-with-eas-build) for building your app
   - [EAS Update](#usage-with-eas-update) for over-the-air updates
3. [Setting up the Sentry-Expo integration](#sentry-integration-with-eas) to view crash reports and session replays directly in your Expo dashboard
```

- Let's update application instances to "app" for consistency
- We use Expo dashboard instead of EAS elsewhere in our docs